docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -110,23 +101,23 @@ A tax provider is ready to establish a connection with the Tax Provider API when
 
 For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) provide an added layer of security for tax providers. They set basic authentication credentials for the tax provider and these basic credentials are used to authenticate each API request to the tax provider from the associated store.
 
-If the tax provider supports all eligible stores, then they may choose to provide an account registration flow in their app iFrame in order to capture these basic authentication credentials from merchants. Learn more about designing the app UI [here](/docs/integrations/apps/guide/ui).
-
-If the tax provider is a private instance, then they may choose to provide the basic authentication credentials themselves.
+Offering an account login flow within the tax provider's app iFrame will commonly be required to capture these basic authentication credentials from merchants. An account registration flow may also be helpful for offering merchants the ability to create a new tax provider account. Learn more about designing the app UI [here](/docs/integrations/apps/guide/ui).
 
-In either case, the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint should be called after the tax provider's app has been successfully installed. Tax providers will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
+In either case, the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint should be called after the tax provider's app has been successfully installed and the merchant's account credentials have been authenticated. Tax providers will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
```

Comment:
We've repeatedly seen scenarios where a tax provider has reached out to support, where the app is installed but a connection has bot been set up. Hopefully this helps guide this area a little further.